src/bundle/Resources/public/js/OnlineEditor/toolbars/config/base-buttons.js

```diff
@@ -0,0 +1,53 @@
+import AlloyEditor from 'alloyeditor';
+
+export default class EzConfigButtonsBase {
+    getButtons(config) {
+        const toolbarConfig = window.eZ.richText.alloyEditor.toolbars[this.name];
+
+        if (!toolbarConfig) {
+            return [];
+        }
+
+        const buttons = [...toolbarConfig.buttons];
+        const attributesEditIndex = buttons.findIndex((button) => button === 'ezattributesedit');
```

Comment:
could be indexOf instead of findIndex
```suggestion
        const attributesEditIndex = buttons.indexOf('ezattributesedit');
```